agent.go

```diff
@@ -859,6 +862,13 @@ func cgroupsMount() error {
 			return err
 		}
 	}
+
+	// Enable memory hierarchical account.
+	// For more information see https://www.kernel.org/doc/Documentation/cgroup-v1/memory.txt
+	if err := ioutil.WriteFile(cgroupMemoryUseHierarchyPath, []byte{'1'}, cgroupMemoryUseHierarchyMode); err != nil {
```

Comment:
Nit: That's a pretty long line. I'd be tempted to perform the `if` test on the line below the write.

Reply:
The best thing to do would be:
```go
return ioutil.WriteFile(cgroupMemoryUseHierarchyPath, []byte{'1'}, cgroupMemoryUseHierarchyMode)
```